### PR TITLE
Add blog post remote-output-service.

### PR DESCRIPTION
**Pull request by coeuvre**

### CODE REVIEW

**Comment by coeuvre** [comment]
cc @EdSchouten

**Review by meisterT — APPROVED** [review]

**Review by EdSchouten — APPROVED** [review]
❤️

**Review by meteorcloudy — APPROVED** [review]

File: _posts/2024-07-23-remote-output-service.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+layout: posts
+title: Introducing Bazel 7.2’s Output Service protocol
+authors:
+  - coeuvre
+  - EdSchouten
+---
+
+## Introduction
+
+One of the exciting new features in Bazel 7.2.0 is support for the Bazel Output Service which allows Bazel to lazily materialize outputs when you access them with normal filesystem operations. This allows you to maintain visibility to the entire output tree while still saving network bandwidth.
```
[the remark this answers is not in the document]
Comment (coeuvre) [reply _posts/2024-07-23-remote-output-service.md]: Done.

**Comment by coeuvre** [comment]
Reviewers: I don't have write permission, please merge it when you think it's ready.

**Pull request merged**